libs/web/shared/ui/src/lib/input/input.props.ts

```diff
@@ -0,0 +1,19 @@
+import type { BorderSize, BorderSizeProps, Height, SizeProps, Width } from '../props';
+
+export const ContainerVariants = ['primary', 'secondary', 'tertiary', 'error'] as const;
+export type ContainerVariant = (typeof ContainerVariants)[number];
+
+export interface TextInputProps extends BorderSizeProps, SizeProps {
+  rounded?: BorderSize;
+  width?: Width;
+  height?: Height;
+  variant?: ContainerVariant;
+  disabled?: boolean;
+  name: string;
+  type: 'text' | 'email' | 'tel' | 'password' | 'url' | 'date';
```

Comment:
would be great if split this type into variable for latter use in the future

Reply:
could you provide `type into variable` example? I'm not sure if I get it right.

Reply:
As a reply to [this comment](https://github.com/deeptech-kmitl/producktivity/pull/42#pullrequestreview-1891062736), I will add `rounded: base` as initial value. If any button needs to use `rounded: full`, it can be config later itself.